examples/basic-tracer-node/README.md

```diff
@@ -57,6 +60,22 @@ Click on the trace to view its details.
 
 <p align="center"><img src="./images/jaeger-ui-detail.png?raw=true"/></p>
 
+### Collector Exporter
+Make sure you have [docker](https://docs.docker.com/) installed
+ - Run the docker container
+   ```sh
+   $ # from this directory
+   $ # open telemetry
+   $ npm run collector:docker:ot
+   $ # or alternatively open census
```

Comment:
I am not sure if we want to add OpenCensus collector.

Reply:
removed